interfaceUtils.py

```diff
@@ -10,28 +10,34 @@
 __all__ = ['requestBuildArea', 'runCommand',
            'setBlock', 'getBlock',
            'placeBlockBatched', 'sendBlocks']
+__author__ = "Nils Gawlik <nilsgawlik@gmx.de>"
+__date__ = "11 March 2021"
 # __version__
+__credits__ = "Nils Gawlick for being awesome and creating the framework" + \
+    "Flashing Blinkenlights for general improvements"
 
 import warnings
 
 import requests
 
 
 class Interface():
-    """**Provides tools for interacting with the HTML interface.**
+    """**Provides tools for interacting with the HTML interface**.
 
     All function parameters and returns are in local coordinates.
     """
 
-    def __init__(self, offset=(0, 0, 0), useBatching=True, batchSize=100):
-        self.buffer = []
+    def __init__(self, offset=(0, 0, 0), buffering=False, bufferlimit=4096):
         self.offset = offset
-        self.useBatching = useBatching
-        self.batchSize = batchSize
+        self.__buffering = False
+        self.bufferlimit = 4096
```

Comment:
I think it's smart to set the default a little bit lower. I would suggest 1024 or 512. This is only slighly slower, but looks better in the game, and less blocks will be un-placed if, for example, the program pauses on an exception during development/debugging